pipenv/cli/command.py

```diff
@@ -62,18 +62,15 @@ def cli(
     state,
     where=False,
     venv=False,
-    rm=False,
-    bare=False,
-    three=False,
-    python=False,
-    help=False,
     py=False,
     envs=False,
```

Comment:
Given that you are removing `--three` and `--python` as explicit arguments here we should add cli smoke tests to make sure they are still parsed properly just in case

Reply:
Will do